src/utils/Sort.ts

```diff
@@ -0,0 +1,5 @@
+export enum Sort {
```

Comment:
It is a good practice to place enums in a separate folder like 'src/enums'. Consider moving 'Sort.ts' to 'src/enums'.